docs/commands/nop.md

```diff
@@ -1,18 +1,22 @@
 ## Command `nop`
 
-The `nop` command allows you to easily skip instructions.
+The `nop` command allows you to easily patch instructions with nops.
 
 ```
-gef ➤  help nop
-Patch the instruction(s) pointed by parameters with NOP. Note: this command is architecture
-aware.
-Syntax: nop [LOCATION] [--nb NUM_BYTES]
-  LOCATION      address/symbol to patch
-    --nb NUM_BYTES      Instead of writing one instruction, patch the specified number of bytes
+nop [LOCATION] [--n NUM_ITEMS] [--b]
```

Comment:
shouldn't it be `NUM_INSTRUCTIONS`?

Reply:
No, because can be instructions or nops so = items